_layouts/gap3.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: start
+---
+<table><tr>
+<td class="gap3bar">This is a page on GAP&nbsp;3, which is
+still available, but no longer supported. The present version is
+GAP&nbsp;4&nbsp;&nbsp;(See&nbsp; <a href="/Gap3/status3.html">
+Status&nbsp;of&nbsp;GAP&nbsp;3)</a>.</td></tr></table>
```

Comment:
I took the liberty of adding this to the layout, and also removed a cache file that was committed by accident